api/jobs/rules.py

```diff
@@ -199,6 +199,9 @@ def create_potential_jobs(db, container, container_type, file_):
                 gear = gears.get_gear_by_name(alg_name)
                 job = Job(str(gear['_id']), inputs, tags=['auto', alg_name])
 
+            if 'config' in rule:
+                job.config = rule['config']
```

Comment:
Will the rule config have the defaults filled out, or will it need to be pushed through `fill_gear_default_values`?

Reply:
The defaults will get filled in `Queue.enqueue_job()`. Let's not use `fill_defaults` when creating a job that will get enqueued through that codepath.